piwik/README.md

```diff
@@ -25,18 +25,20 @@ your ~friendica server and then enter the needed details on the config page
 for the addon.
```

Comment:
Could you please fix the format of this file as well to the one-sentence-per-line format.